frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -305,6 +312,10 @@ const HostSummary = ({
         statusText = diskEncryptionEnabled || "Unknown";
     }
 
+    if (isIosOrIpadosHost) {
```

Comment:
can we just OR this with the condition on line 290, or does this specifically have to be `null` and not `<></>`?